ios-base/scripts/format.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+git diff --name-only | grep -e '\(.*\).swift$' | while read filename; do
+  # ref: https://github.com/yonaskolb/Mint/issues/112
+  mint run swiftformat swiftformat "$(git rev-parse --show-toplevel)/$filename"
```

Comment:
nits.
use `$SRCROOT/../$filename` instead of `$(git rev-parse --show-toplevel)/$filename` is also ok 😄

Reply:
I'm not sure which is better 🤔

Reply:
If we match writing style about this, I think you should use `$SRCROOT/../$filename`, because it is already used by `lint.sh`. 👍

Reply:
I think `$(git rev-parse --show-toplevel)/$filename` better than `$SRCROOT/../$filename`. If script using `$(git rev-parse --show-toplevel)/$filename`, it's executable via commandline.

However, since it is rarely executed from the command line, I think that either is good.

Reply:
Thank you both! I decided to use `$SRCROOT` to match others.